src/plugins/discover/public/application/components/create_discover_grid_directive.tsx

```diff
@@ -21,6 +22,11 @@ export function DiscoverGridEmbeddable(props: DiscoverGridProps) {
 
   return (
     <I18nProvider>
+      {props.isLoading ? (
+        <EuiProgress size="xs" color="accent" />
+      ) : (
+        <div className="dscEmbeddableProgressInactive" />
```

Comment:
I've tried to work with `position="absolute"` in `EuiProgress` however, then I needed to wrap this and the grid with a relative positioned `<div>`, and this messed up vertical sizing of data grid. so I've decided to render an inactive progress bar (grey background so it's invicible)

Reply:
I didn't even notice the jumping!

Reply:
it was 2px. it's jumping when you use a small interval